src/main/scala/mesosphere/marathon/core/group/impl/GroupManagerImpl.scala

```diff
@@ -49,6 +50,7 @@ class GroupManagerImpl(
   private[this] val root = LockedVar(initialRoot)
 
   private[this] val dismissedDeploymentsMetric = Metrics.counter(ServiceMetric, getClass, "dismissedDeployments")
+  private[this] val groupUpdateSizeMetric = Metrics.minMaxCounter(ServiceMetric, getClass, "queueLength")
```

Comment:
"queueLength"...  what do you think of  "queueSize" or "currentQueueSize"?
If we want to be consistent with deployments we would call it "currentQueueCount"

Reply:
queueSize is my favorite ... thanks guys!